pom.xml

```diff
@@ -62,12 +77,38 @@
                 </executions>
                 <configuration>
                     <configLocation>${maven.checkstyle.plugin.configLocation}</configLocation>
+                    <sourceDirectories>${project.build.sourceDirectory}</sourceDirectories>
```

Comment:
What is this property value? /src/main/java? You should add test for checkstyle as well